README.md

```diff
@@ -22,26 +23,41 @@ Here is a basic tutorial introduction to monadic parsing with this package.
 
 ### Parsers
 
-A parser turns a string into a data structure. Parsers in this library have the type `Parser s a`, where `s` is the type of the input string, and `a` is the type of the data which the parser will produce on success. `Parser s a` is a monad. It’s defined in the module `Text.Parsing.Parser`.
+A parser turns a string into a data structure. Parsers in this library have the type `Parser s a`, where `s` is the type of the input string, and `a` is the type of the data which the parser will produce on success. `Parser s` is a monad. It’s defined in the module `Text.Parsing.Parser`.
 
-Monads can be used to provide context for a computation, and that’s how we use them in monadic parsing. The context provided by the `Parser` monad is *the parser’s current location in the input string*. Parsing starts at the beginning of the input string.
+Monads can be used to provide context for a computation, and that’s how we use them in monadic parsing.
+The context provided by the `Parser s` monad is __the parser’s current location in the input string__.
+Parsing starts at the beginning of the input string.
 
-Parsing requires two more capabilities: *choice* and *failure*.
+Parsing requires two more capabilities: __alternative__ and __failure__.
```

Comment:
The "choice" -> "alternative" change makes the language fairly awkward. What was the reason for changing it?

Reply:
Because “alternative” is the term used by the `Alt` typeclass. For clarity I like to try to always use the same word when I'm talking about the same thing.

Reply:
I think "choice" is a more familiar word to use here because you clarify what you mean by "choice" in the next few paragraphs: `Alt`.